node/clients/StorefrontPermissions.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { InstanceOptions, IOContext } from '@vtex/api'
+import { AppGraphQLClient } from '@vtex/api'
+
+import { QUERIES } from '../resolvers/Routes/queries'
+import { createHeaderWithToken } from './index'
+
+export default class StorefrontPermissions extends AppGraphQLClient {
+  constructor(ctx: IOContext, options?: InstanceOptions) {
+    super('vtex.storefront-permissions@1.x', ctx, options)
+  }
+
+  public checkUserPermission = async (): Promise<any> => {
+    return this.graphql.query(
+      {
+        extensions: {
+          persistedQuery: {
+            provider: 'vtex.storefront-permissions@1.x',
+            sender: 'vtex.b2b-organizations@0.x',
+          },
+        },
+        query: QUERIES.getPermission,
+        variables: {},
+      },
+      {
+        params: {
+          headers: createHeaderWithToken(this.context),
+          locale: this.context.locale,
```

Comment:
It doesn't seem like this locale was being sent before, and it's not being sent in the Organizations client queries. Can you explain why add it only here?

Reply:
It was implemented previously [here](https://github.com/vtex-apps/b2b-checkout-settings/blob/master/node/clients/graphqlServer.ts#L17). I don't know why, but I just found the use in storefront-permissions, like the [client in b2b-organization-graphql](https://github.com/vtex-apps/b2b-organizations-graphql/blob/master/node/clients/storefrontPermissions.ts#L398).

Reply:
Got it. It used to pass the session token as well, right? That part might be important for these requests in some cases, maybe safer to keep it in both of the new clients, so that the behavior there is the same?

Reply:
Ok, I can put for in the `b2b-organization-graphql` client but the app implementation doesn't use it.